src/web/pages/nvts/row.jsx

```diff
@@ -102,14 +104,19 @@ const Row = ({
       <TableData align="end">
         {entity.qod && <Qod value={entity.qod.value} />}
       </TableData>
-      <TableData>
-        {isNumber(entity?.epss?.max_severity?.score)
-          ? entity.epss?.max_severity?.score.toFixed(5) : _("N/A")}
-      </TableData>
-      <TableData>
-        {isNumber(entity?.epss?.max_severity?.percentile)
-          ? entity.epss?.max_severity?.percentile.toFixed(5) : _("N/A")}
-      </TableData>
+      {
+        gmp.settings.enableEPSS &&
+        <>
+          <TableData>
+            {isNumber(entity?.epss?.max_severity?.score)
```

Comment:
Question: Not sure what convention we have in the codebase for in lining, but maybe if we move it to a `const` could be easier to understand with the name?